src/share/classes/com/sun/javatest/regtest/tool/i18n.properties

```diff
@@ -294,6 +294,15 @@ help.pool.pool_idle_timeout.arg=<number>
 help.pool.pool_idle_timeout.desc=\
      The time, in seconds, before an idle VM in the pool is \
      automatically closed.
+help.pool.agent_attempts.arg=<number>
+help.pool.agent_attempts.desc=\
+    The maximum number of times the pool will be queried for a agent VM \
+    when running an action. A value of 1 will mean that no retry will be \
+    attempted if either a new agent VM creation fails or an existing one \
+    couldn't be obtained from the pool. Similarly, a value of 2 will imply \
+    that if the agent VM creation fails or an existing one couldn't be \
+    obtained from the pool, then 1 retry will be attempted. \
```

Comment:
The text about "queries" seems "weird".  Suggest:

The number of attempts jtreg will make to access an agent with the desired characteristics, either by creating a new one, or obtaining one from the pool of reusable VMs. The minimum value, and the default value, is 1.

Reply:
Thank you for providing this text. I agree it's much more concise and provides all the necessary details. I've updated the PR to use this text.